ztests/suite/zqd/rec-curl.yaml

```diff
@@ -0,0 +1,44 @@
+script: |
```

Comment:
For this file and the others under suite/zqd, would you please use `recruiter` instead of `rec` in their file names?